plugins/inventory/now.py

```diff
@@ -261,7 +265,9 @@ def parse(self, inventory, loader, path,
         selection = self.get_option('selection_order')
         fields = self.get_option('fields')
         table = self.get_option('table')
-        filter_results = self.get_option('filter_results')
+        filter_results = os.environ.get('SN_FILTER_RESULTS')
+        if not filter_results:
+            filter_results = self.get_option('filter_results')
```

Comment:
`self.get_option` will look up from the environment variable and fall back to the option specified in the inventory file. The direct env lookup and if condition are redundant behavior.

Reply:
> `self.get_option` will look up from the environment variable and fall back to the option specified in the inventory file. The direct env lookup and if condition are redundant behavior.

Hi
I tried setting the filter via filter_results environment variable and not specify a filter_results in now.yml
export filter_results="somefilter"
ansible-inventory -i now.yml  --graph -vvv
When I run ansible-inventory, no filter is applied and I can see that sysparm_query=  is empty when running ansible-inventory in verbose mode (i tried this with ansible-2.9.20).